adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -49,13 +50,14 @@ def __init__(self, esp, secrets, status_pixel=None, attempts=2):
         """
         # Read the settings
         self.esp = esp
-        self.debug = False
+        self.debug = debug
         self.ssid = secrets['ssid']
         self.password = secrets['password']
         self.attempts = attempts
         requests.set_interface(self.esp)
         self.statuspix = status_pixel
         self.pixel_status(0)
+    # pylint: enable=too-many-arguments
```

Comment:
extra pylint directive here?

Reply:
I don't think so, unless i'm misunderstanding the question. I disabled `too-many`arguments on line 41 because I updated __init__ to accept the debug boolean, pushing it over the pylint suggested arg limit, so I'm re-enabling that linting rule after the __init__ function so it applies to every other method in this file.

Reply:
This is fine, you're understanding correctly.